Funcky.Analyzer/Funcky.Analyzer/Funcky.Analyzer.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk; Microsoft.Build.CentralPackageVersions">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <IsPackable>false</IsPackable>
+
+        <!-- Avoid ID conflicts with the package project. -->
```

Comment:
Why is there a conflict when packing is disabled for this project?

Reply:
Auto generated